sequence_python/models/ir_sequence.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2020 Sunflower IT (<https://sunflowerweb.nl>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+import logging
+import random
+import string
+import uuid
+
+from odoo import fields, models
+from odoo.tools import safe_eval
+
+_logger = logging.getLogger(__name__)
+
+
+DEFAULT_PYTHON_CODE = "number_padded"
+
+
+class IrSequence(models.Model):
+    """
+    Inherit standard ir.sequence to let the possibility of using a Python formula
+    to calculate the sequence from input variables such as the sequence number.
+    This allows obfuscation of the order in which sequences are given out,
+    but can also be used for any other purpose.
+    """
+
+    _inherit = "ir.sequence"
+
+    # Python code
+    use_python_code = fields.Boolean(string="Use Python", default=False)
+    python_code = fields.Text(
+        string="Python expression",
+        groups="base.group_system",
```

Comment:
I think this geoup is unnecessary. Users that can create sequence should use this feature